sdk/marketplace/Microsoft.Azure.Management.Marketplace/src/Microsoft.Azure.Management.Marketplace.csproj

```
@@ -6,9 +6,9 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Marketplace</PackageId>
     <Description>Microsoft Azure Marketplace Library</Description>
-    <Version>1.0.1</Version>
+    <Version>1.0.2</Version>
```

For feature addition we suggest incrementing the minor version. The patch version is to be updated only when  bug fixing.
```suggestion
    <Version>1.1.0</Version>
```